specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+This is the specified format for providing data in the form of key-value pairs, which are also known as name-value pairs, attribute-value pairs, and field-value pairs in other contexts. 
```

Comment:
For consistency, you might consider adding a sentence like we do for other format attributes where we explain why this would be helpful to consumers.